share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
@moollaza There is another problem with using `subtitle`.

Using `subtitle`:
![selection_140](https://cloud.githubusercontent.com/assets/5206277/16269979/3085f1b2-38b2-11e6-8759-0c7541668389.png)
Using `description`:
![selection_139](https://cloud.githubusercontent.com/assets/5206277/16269998/464e683a-38b2-11e6-925c-480f37218d20.png)

I will rather write CSS for removing the extra space using `description`. What do you think?